### ISSUE TITLE
Handle JGrasp downloads better

### ISSUE THREAD

**Issue by kylelaker** [jmunixusers/cs-vm-build#116]
Use the solution @ripleymj proposed in #112

**Comment by kylelaker** [comment]
For completeness, copying the suggestion here.

> [W]e should see if we can POST to their download form: http://spider.eng.auburn.edu/user-cgi/grasp/grasp.pl and chase the redirect it serves to the latest version

**Comment by ripleymj** [comment]
On 2nd review, this isn't that simple. That POST mechanism won't tell us if the installed version needs to be replaced.

**Comment by kylelaker** [comment]
True. Currently we rely on the hash to determine when to pull a new version and I am hesitant to significantly increase the logic of that play to do any sort of version determination.
